vale/code/arch/x64/interop/Vale.Wrapper.X64.Sha.fsti

```diff
@@ -6,17 +6,19 @@ module B = LowStar.Buffer
 module IB = LowStar.ImmutableBuffer
 module BV = LowStar.BufferView
 module HS = FStar.HyperStack
+module BF = Vale.Arch.BufferFriend
 open FStar.Mul
 open Vale.SHA.SHA_helpers
+open Lib.IntTypes
 
 unfold
-let uint32_p = B.buffer UInt32.t
+let uint32_p = B.buffer uint32
 unfold
-let uint32_i = IB.ibuffer UInt32.t
+let uint32_i = IB.ibuffer uint32
 unfold
-let uint8_p = B.buffer UInt8.t
+let uint8_p = B.buffer uint8
 unfold
-let uint64 = UInt64.t
+let uint64 = uint_t U64 PUB
```

Comment:
This is `PUB` while the types above (`uint32, uint8`) are `SEC`. Why the difference?

Reply:
This uint64 will only be used to represent lengths of buffers, which will be passed as public arguments to Vale. I'd be fine with renaming it uint64_pub to avoid confusion in this fsti